bin/tools/ocamlformat.ml

```diff
@@ -37,25 +37,47 @@ let build_dev_tool common =
      | Ok () -> ())
 ;;
 
-let term =
-  let+ builder = Common.Builder.term
-  and+ args = Arg.(value & pos_all string [] (info [] ~docv:"ARGS")) in
+let go builder command =
   let common, config = Common.init builder in
   Scheduler.go ~common ~config (fun () ->
     let open Fiber.O in
     let* () = Lock_dev_tool.lock_ocamlformat () |> Memo.run in
     let+ () = build_dev_tool common in
-    run_dev_tool (Common.root common) ~args)
+    match command with
+    | `Run_with_args args -> run_dev_tool (Common.root common) ~args
+    | `Install -> ())
```

Comment:
It seems strange to me that there would be a function `go` that doesn't do anything in some cases. In that case I think it would make more sense to refactor it in some way where it is never called.